src/api/controllers/userController.ts

```diff
@@ -90,14 +90,24 @@ export class UserController {
 
     const resultLength = user?.length ?? 0;
     if (resultLength < 1) {
-      res.sendStatus(404);
+      errorLogger.error(
+        `No results received when loading user entry for ${req.user?.email}`,
+        traceId
+      );
+      res.status(404).json({
+        errorHash: traceId,
```

Comment:
It would be nice to have a pattern where we wouldn't have to explicitly include the traceId in every route handler response that needs it. No need to do that in this PR, but it's worth thinking about whether there's a good way.

Reply:
yeah I'm hoping the DI work will help with this, but haven't fully thought it through yet.